### PR TITLE
Adding debug logs + Adding liquibase config

### PR DESCRIPTION
**Pull request by eladiw**
Adding some debug logs across the services + moved liquibase config to service itself (role).

### Checklist:

- [x] Branch name follows convention (feature/arts-#-short-name OR fix/arts-#-short-name)
- [x] I have included a short-meaningful title, description and changes for this PR

### CODE REVIEW

File: init-service/src/main/java/uk/ac/ox/ndph/mts/init_service/loader/Loader.java [hunk 1]
```diff
@@ -13,6 +15,7 @@
 
 @Component
 public class Loader implements CommandLineRunner {
+    private final Logger logger = LoggerFactory.getLogger(CommandLineRunner.class);
```
Comment (darrengsmithson) [inline init-service/src/main/java/uk/ac/ox/ndph/mts/init_service/loader/Loader.java:18]: use the more standard approach - private static final Logger LOGGER ?? Theres a little bit of a mix in the codebase tbf..

**Review by darrengsmithson — COMMENTED** [review]
PR title?

**Review by darrengsmithson — COMMENTED** [review]
I dont think the liquibase properties should be in the bootstrap file - the bootstrap should be used for loading configuration properties from external sources. I also think we're masking the real issue here - shouldnt we be always making the config server be available B4 other services start? Even if we workaround this issue in this service for this problem what if we need to config server to start 1st for another service - do we then 'move' those config properties to the bootstrap and thus the bootstrap becomes a fail-over option for non related properties??

File: init-service/src/main/java/uk/ac/ox/ndph/mts/init_service/loader/Loader.java (hunk 1 above)
Comment (eladiw) [reply init-service/src/main/java/uk/ac/ox/ndph/mts/init_service/loader/Loader.java:18]: I moved everything to non  static, all except places that must have static loggers (i.e. static methods)

**Review by LizaShak — APPROVED** [review]

**Pull request closed without merging**